fetch_submissions.py

```diff
@@ -140,14 +143,21 @@ def save_submission_as_file(submission):
     outfile.write(submission['code'])
 
 
+def get_user_credentials():
```

Comment:
I think we should stick with this function name but not have it return a boolean. Instead, the function could invoke `sys.exit(1)` instead of the `__main__`. This ties in with my comment on a seperate function for login.